framework/tlvf/yaml/tlvf/test/tlvVarList.yaml

```diff
@@ -38,3 +38,7 @@ cInner:
     _type: uint8_t
     _length: [ list_length ]
   var1: uint32_t
+  unknown_length_list:
```

Comment:
Could you give a different name than the unkown_length_list that already exists in the outer class? Otherwise the code gets a little bit confusing.

The same problem already exists for var1, with things like `var1()->var1()` in the code...